docs/introduction.rst

```diff
@@ -1,73 +1,47 @@
 
-Data items represent data arrays with anywhere from one to five dimensions. The data array associated with a data item can be 1D or 2D data and then organized into 1D or 2D collections or 1D sequences.
+The utility panels surrounding the central workspace allow you to make adjustments to a selected data item in a display panel. For more information on their specific functions, see :ref:`user-interface`.
 
-A spectrum image would be a 2D collection of 1D data. This would be typical data acquired from a STEM microscope spectrum imaging acquisition.
+Data and Display
+================
+Data is stored in a project as data items. Data items can be dragged into display panels to be displayed. Data items that are displayed in a display panel can be manipulated with graphics, computations, and more. These displayed data items are referred to as display items. Display items can be modified and viewed to see various aspects or values of the original data item. 
 
-Data Items are displayed using one or more *Display Items*.
+Read more in :ref:`data-items`.
 
-See :ref:`data-items`.
+Graphics and Masks
+==================
+Nion Swift provides plenty of useful graphics to narrow down your view of the data. Use graphics to highlight portions of the data, select areas for processing, or to focus on specific sections of data.
 
-Projects
---------
-Data items are organized into projects. Projects are stored on disk and take the form of an index file and an associated data directory.
+Masking allows you to block out the rest of an image and manipulate only the data you want. Choose from a variety of mask shapes to precisely define your area of interest.
 
-You can have any number of projects, but only a single one can be open at once. When you open a project, it is associated with a main document window.
+See :ref:`graphics` for more information.
 
-See :ref:`data-management`.
+Processing and Analysis
+=======================
+Processing data by adding line profiles to an image, running Python scripts, filtering data, etc. can give you deeper insight into your data. Use the :ref:`inspector panel` to adjust various settings to tailor processes to your specific needs. Customizing processes allows for easier and more specific analysis of data.
 
-Workspaces
-----------
-The main window is organized around a main area called a workspace. A workspace has a layout which includes one or more display panels in which display items can be visualized.
+See :ref:`processing` for more information.
 
-You can create additional workspace for different uses and easily switch between them.
+Projects and File Management
+============================
+Imported and gathered data items are stored as a project in the form of an index file on your computer. Projects can be transferred just like any other file and can be opened in other installations of Nion Swift provided the versions are compatible.
```

Comment:
This needs a bit of rewording. My suggestion is to start with the key point:

"Data is organized into projects. A project file..."

Also, what do you think about putting this earlier in the document? I'm not sure where it fits; but it's something to think about.

Reply:
Reiterating this comment.

The old text said "Data items are organized into projects. Projects are stored on disk and take the form of an index file and an associated data directory."

I would keep it simple but reword something like this:

"Nion Swift organizes data items into projects. Projects are folders/directories on your computer which contain both an index file for tracking relationships between data items and the individual files for each data item. Projects can be archived and/or moved to other computers and opened in other installations of Nion Swift."

Overall, this may take a few passes to get right, but I think this is a good starting paragraph.

Also, I don't want to lose this key piece of information:

"You can have any number of projects, but only a single one can be open at once. When you open a project, it is associated with a main document window."